codersdk/workspacesdk/workspaceagentconn.go

```diff
@@ -1,4 +1,4 @@
-package codersdk
+package workspacesdk
```

Comment:
Thoughts on merging this into `workspaceagents.go` and then renaming that file to `workspacesdk.go`?

Reply:
Personally I prefer them split up, since them together would be a file >1k lines

Reply:
Disagree but fine

Reply:
I rearranged it a bit